app/services/utils/collection.py

```diff
@@ -0,0 +1,52 @@
+from typing import Optional, Dict, Any
+
+import requests
+from requests import RequestException
+
+from app.utils import config
+
+settings = config.get_settings()
+
+
+def get_collection_items_url(collection_id: str) -> Optional[str]:
+    url = settings.stac_url + collection_id
+    response = requests.get(url)
+    response.raise_for_status()
+    collection_data = response.json()
+
+    if not collection_data:
+        raise ValueError(f"No collection found at URL: {url}")
+
+    return url
```

Comment:
at this moment the function is returning the collection url not the items url. You can either change the function name or check for the items in this same function and return the items url